packages/documentation-site/patternfly-docs/content/design-guidelines/styles/motion/motion.md

```diff
@@ -0,0 +1,154 @@
+
+1. **Easing:** Describes how an element transitions over time.
+
+1. **Properties:** The characteristics of the element being transitioned. Animations may have multiple properties, including: 
+    - Fade in/out
+    - Scale
+    - Rotate
+    - Color transitions
+    - Slide in/out
+    - Expand/collapse
+
+## Developing and designing with motion 
+
+### Figma 
+In Figma, you can find annotations and prototype videos for each animation...
+
+### Tokens
+
+PatternFly component animations are created using design tokens that cover the different elements of an animation, including duration, delay, and timing. Like we do with color tokens, we implement motion by using semantic tokens, which are built off of base tokens.
+
+If you aren't familiar with our token system, [refer to this overview](/tokens/about-tokens).
+
+#### Duration 
+Duration tokens specify the length of time that different animation should take to complete.
+
+| Animation | Tokens | Length |
+| --- | --- | --- |
+| Slide-out | -pf--t--global--motion--duration--slide-out--short<br />-pf--t--global--motion--duration--slide-out--default<br />-pf--t--global--motion--duration--slide-out--long | 300<br />400<br />500
+| Slide-in |  |
+| Fade |  |
+| Icon |  |
+<br />
+
+#### Delay 
+Delay tokens specify the length of time that should pass before an animation begins. Delay options include none, short, default, and long.
+
+#### Timing 
+Timing-function tokens specify the easing path that an animation takes. These paths are defined by cubic bezier curves, which define the start and end points ... [fact check]
+
+Options include: 
+- **Accelerate:** Use to specify when an animation should begin to speed up.
+- **Default:** Use to specify when an animation should maintain a cosistent speed.
+- **Decelerate:** Use to specify when an animation should begin to slow down.
+
+### Motion patterns
+
+A **motion pattern** is a combination of tokens and properties that work together to solve common animation needs. These patterns help ensure that the behavior of a type of animation is consistent across similar use cases—even if different components are involved.
+
+For example, an "action fade" pattern may dictate how animation is applied when a user selects an action. This pattern specifies the timing and duration of the fade, so that the animation is the same across different action components. In this example, we would want users to see the same hover/click animation whether they're interacting with an action button, an action menu, and so on.
+
+### Custom motion 
+
+For some scenarios, like creating a new [PatternFly extension](/extensions/about-extensions), you may need to customize motion that doesn't already exist in PatternFly components. When you're creating a new animation behavior, make sure that:
+- There is no existing support for the animation.
+- The animation adheres to our motion principles. 
+- Your implementation uses appropriate semantic tokens. 
```

Comment:
Add that the user's setting for `prefers-reduced-motion` should be respected? Unless that is expected as part of adhering to our motion principles.